src/main/java/org/ow2/proactive/connector/iaas/cloud/provider/azure/scaleset/AzureScaleSetProvider.java

```diff
@@ -0,0 +1,327 @@
+                                      // assign a public IP address to the load balancer
+                                      .definePublicFrontend(azureFrontendName)
+                                      .withExistingPublicIpAddress(publicIPAddress)
+                                      .attach()
+                                      // Add one backend address pools
+                                      .defineBackend(azureBackendPoolName)
+                                      .attach()
+                                      // Define mock TCP probe and HTTP rule
+                                      .defineTcpProbe(azureTCPProbeName)
+                                      .withPort(22)
+                                      .withNumberOfProbes(1)
+                                      .attach()
+                                      .defineLoadBalancingRule(azureHTTPLoadBalancingRule)
+                                      .withProtocol(TransportProtocol.TCP)
+                                      .withFrontend(azureFrontendName)
+                                      .withFrontendPort(81)
+                                      .withProbe(azureTCPProbeName)
+                                      .withBackend(azureBackendPoolName)
+                                      .attach()
+                                      // Define NAT pools for SSH, PNP and PNPS
+                                      .defineInboundNatPool(azureNATPoolSSHName)
+                                      .withProtocol(TransportProtocol.TCP)
+                                      .withFrontend(azureFrontendName)
+                                      .withFrontendPortRange(50000, 50099)
+                                      .withBackendPort(22) // SSH
+                                      .attach()
+
+                                      .defineInboundNatPool(azureNATPoolPNPName)
+                                      .withProtocol(TransportProtocol.TCP)
+                                      .withFrontend(azureFrontendName)
+                                      .withFrontendPortRange(30000, 30099)
+                                      .withBackendPort(64738) // PNP
+                                      .attach()
+                                      .defineInboundNatPool(azureNATPoolPNPSName)
+                                      .withProtocol(TransportProtocol.TCP)
+                                      .withFrontend(azureFrontendName)
+                                      .withFrontendPortRange(40000, 40099)
+                                      .withBackendPort(64739) // PNPS
+                                      .attach()
+                                      .create();
+
+        // Create the Scale Set (multi-stages)
+        VirtualMachineScaleSet.DefinitionStages.WithLinuxRootPasswordOrPublicKeyManagedOrUnmanaged virtualMachineScaleSet_stage1 = azureService.virtualMachineScaleSets()
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this local variable name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS00117)